README.md

```diff
@@ -27,12 +27,25 @@ DB_PASS=my-database-password
 JWT_SECRET=secret-key
 COOKIES_EXPIRES_MS=cookies-expires-ms
 JWT_EXPIRES_MS=jwt-expires-ms
+REDIS_URL = redis-url
 ```
 
 You can then use the `connectDB` and `disconnectDB` functions provided in the
 `/server/utils/database.js` to both connect and disconnect from the database
 using your personal login information from the `.env` file.
 
+### Redis
+
+### Setup
+
+To set-up the redis locally, you need to follow the below link.
```

Comment:
ok

Reply:
Thank you 😺
In terms of the Markdown best-practices, it's recommended not have links "exposed" (just plain links without being associated with some test):

```markdown
So instead of:
Here's the code: https://www.example.com

Add brackets around the text, followed immediately by parenthesis around the URL (there can't be an empty space between the closing bracket and the opening parenthesis):
[Here's the code](https://www.example.com)
Or 
Here's the [code](https://www.example.com)
```

The syntax/grammar changes are:

- Instead of saying "below link", it's more appropriate in this context to say "link below" (and if you're adding the Markdown best-practice from above, you could even say "... the following [link](...)."
- On line 44, I would suggest the following changes (my edits are in bold): "And if you have **a** windows machine and "Enable comm**a**nd doesn't work" Please follow **the instructions in this** youtube video." (and you can make the "this" into a Markdown link.)

I hope you're having a good weekend! 😃